README.md

```diff
@@ -6,6 +6,7 @@
   - [Organization](#approaches-to-project-organization)
   - [DTR](#dtrs)
   - [Contributors](#contributors)
+  - [Retro](#retro)
```

Comment:
nit pick: table of contents should be in same order as the sections. We're also missing a section.
But since the table of contents is a fancy add-on anyway, we can add this to our "Unimportant/do it if there's time" section of Miro board

Reply:
I like this